build.gradle.kts

```diff
@@ -32,7 +32,8 @@ googleJavaFormat {
 
 dependencies {
     api("io.micrometer:micrometer-core:1.3.3")
-    api("com.newrelic.telemetry:telemetry:0.4.0")
+    api("com.newrelic.telemetry:telemetry:0.6.1")
+    api("org.slf4j:slf4j-api:1.7.30")
```

Comment:
What's the motivation for this change? I'm only concerned with pinning the version here as a `api` declaration and what impact that could have on consumers of the library.

Reply:
I think that previously it must have been picked up as transitive from the SDK before or something, because as soon as I upgraded to `0.6.1` it broke compilation.  So I added a dependency here on slf4j directly (since it is used in the registry).  

What else can we do?

Reply:
Can we make it `implementation`, rather than `api`?

Reply:
Sure thing.